conf/cassandra.yaml

```diff
@@ -441,34 +441,36 @@ counter_cache_save_period: 7200
 # saved_caches_directory: /var/lib/cassandra/saved_caches
 
 # Number of seconds the server will wait for each cache (row, key, etc ...) to load while starting
-# the Cassandra process. Setting this to a negative value is equivalent to disabling all cache loading on startup
+# the Cassandra process. Setting this to 0 is equivalent to disabling all cache loading on startup
```

Comment:
This is an interesting one. It's not explicitly enforced that 0 disables cache loading, but that's how it works in `AutoSavingCache`, at least for now.

Reply:
Ha...this is in the yaml so there might be users which use negative... I will have to take care of converter I guess. Good point.

Reply:
BUT it is parameter added recently post-4.0 so we don't need backward compatibility :-)